docs/develop/api-reference/index.mdx

```diff
@@ -485,6 +485,28 @@ The resulting incremented value must be between -9223372036854775808 and 9223372
 
 :::
 
+### SetIfNotExists
+
+Associates the given key with the given value. If a value for the key is already present it is not replaced with the new value.
```

Comment:
This reads funny to me since you use 'given' twice in the same sentence. I suggest modifying it to 

```
Associates the provided value to a cache item with a given key.
```